newsletters/2015-12-25-28.md

```diff
@@ -0,0 +1,54 @@
+---
+published: true
+subject: Meatless meatballs, electricity from cheese,
+teaser: 🐄🌞🚀
+background: 2015-12-12-26-party.gif  
+---
+# Hello!
+
+This week an issue all about energy -- and a big rocket!
+
+* Using leftover cheese 🐄to power an entire village?
+* Cleaning the water with rubbish bins
+* A 'new' way to capture the energy of the sun
+* SpaceX's successful landing of the Falcon rocket 🚀
+
+![cheese](2015-12-25-28-cheese.jpg)
+
+Any idea how much potential energy is thrown with the garbage by cheese companies? Apparently enough to power an entire village!
+
+This cheese manufacturer decided to partner up with the local power plant to use excess skimmed whey -- an unneeded by-product of making the cheese -- as power for 1500 homes. How cool 😎is that?
+
+The energy comes from the bacteria that find the whey delicious and produce methane, a biogas, in return. Thanks little guys!
+
+[eddie.net](http://www.edie.net/news/5/French-power-station-generates-electricity-for-1-500-from-cheese/)
+
+![rubbishbin](2015-12-25-28-bin.jpg)
+
+There is a lot of waste in the oceans -- continent sized collections of small plastics and lot's of oil. These surfer guys bumped into so much waste while surfing that they decided to create 'rubbish bins' to catch the it!
```

Comment:
Maybe remove that their is so much waste in the ocean people no this by now...

Yo can keep the part that they;re surfers and they bumped in to it though